core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -67,24 +78,35 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
     producer.close(60, TimeUnit.SECONDS)
   }
 
+  def produceMessages(topic: String, range: Range) = {
+    Source(range)
```

Comment:
I tried to use Sink.ignore as follows
```
 def produceMessages(topic: String, range: Range) = {
    Source(range)
      .map(n => new ProducerRecord(topic, partition0, null: Array[Byte], n.toString))
      .via(Producer.flow(producerSettings))
      .toMat(Sink.ignore)(Keep.right)
```

however the types don't work out.  The error is found ```Flow``` required ```Graph```.